### DISCUSSION TITLE
What are you looking for?

### DISCUSSION THREAD

**Discussion by ddjerqq** [community/community#147399] [Other Feature Feedback, Questions, & Ideas]
.

**Comment by davevad93** [comment]
Hi @ddjerqq ,

This is what the [GitHub Username Policy](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy) states:

> Keep in mind that not all activity on GitHub is publicly visible; accounts with no visible activity may be in active use.
>
> If the username you want has already been claimed, consider other names or unique variations. Using a number, hyphen, or an alternative spelling might help you identify a desirable username that's still available.

In fact, if you try to contact [support](https://support.github.com/contact/account) about this issue, selecting `Dormant, inactive, or squatting username` will give you the following:

![0000000000000000000](https://github.com/user-attachments/assets/f4ebf50c-dd26-4ea2-ae97-477b5003e652)

Therefore, unless there's a violation of the [Trademark Policy](https://docs.github.com/en/site-policy/content-removal-policies/github-trademark-policy), it's not possible to reclaim an already used username because, as stated above, "**accounts with no visible activity may be in active use**".

**Reply by ddjerqq** [reply]
I asked for help with regards to contacting this person, not to just me and GitHub take their username without their usage. so me and them could negotiate this together with or without GitHub's involvement

**Reply by davevad93** [reply]
I see @ddjerqq , in this case you should check if this person's profile has any contact information in their bio, another way would be to check their commit history to find out their email, but as you said there's no visible activity at all, so my only suggestion to try to contact this person would be to open a ticket [here](https://support.github.com/contact/account), select `Other` and ask GitHub support if anything can be done about it.

**Reply by ddjerqq** [reply]
they indeed have nothing, all clear profile, not even a single commit. thank you very much though!

**Reply by davevad93** [reply]
You're welcome @ddjerqq 👍

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @ddjerqq!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!

**Reply by ddjerqq** [reply]
I already solved it thanks
